CMakeLists.txt

```diff
@@ -43,8 +43,10 @@ set(project_version "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT
 if(NOT DEFINED PROJECT_VERSION_TWEAK)
 	set(project_version "${project_version}")
 elseif(PROJECT_VERSION_TWEAK EQUAL 0)
+	set(debian_version "${project_version}~dev-1")
 	set(project_version "${project_version}-dev")
 elseif(PROJECT_VERSION_TWEAK GREATER 1)
+	set(debian_version "${project_version}~pre.${PROJECT_VERSION_TWEAK}-1")
```

Comment:
It is not entirely clear for me what this `-1` means and why it is hardcoded.

Reply:
To comply with the debian version policy.

Reply:
Can it be 2, 3 etc in future?

Reply:
Unlikely, so far we haven't had any reason to change it. In case we need to change in a further date I can make it default to `-1` and allow the user to set a value.